themes/classic/_dev/js/customer.js

```diff
@@ -0,0 +1,18 @@
+import $ from 'jquery';
+
+function initRmaItemSelector () {
+  $('#order-return-form table thead input[type=checkbox]').on('click', function () {
+    var checked = $(this).prop('checked');
+    $('#order-return-form table tbody input[type=checkbox]').each(function (_, checkbox) {
+      $(checkbox).prop('checked', checked);
+    })
```

Comment:
And btw we're using ES6 so you can have fun with [new language features](https://github.com/DrkSephy/es6-cheatsheet).

E.g.:

```javascript
function initRmaItemSelector () {
  $('#order-return-form table thead input[type=checkbox]').on('click', () => {
    const checked = $(this).prop('checked');
    $('#order-return-form table tbody input[type=checkbox]').each((_, checkbox) => {
    $(checkbox).prop('checked', checked);
  })
```

It doesn't add much value here, but some things are very helpful :)